frontend/apps/mobile/src/app/pages/page-manage-tournament/page-manage-tournament.component.ts

```diff
@@ -0,0 +1,111 @@
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component, inject,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+import { Subject, takeUntil } from 'rxjs';
+
+import { TeamDataService } from '@jtr/business-domain/team';
+import { TournamentDataService } from '@jtr/business-domain/tournament';
+import { EmailRecipientEnum } from '@jtr/data-domain/tournament-data';
+
+import {
+  ButtonColorEnum,
+  ButtonComponent,
+  ButtonFunctionType,
+  ButtonTypeEnum,
+  EditorComponent,
+} from '../../ui-shared';
+import { TranslatePipe } from '@ngx-translate/core';
+import { InputTextModule } from 'primeng/inputtext';
+
+@Component({
+  standalone: true,
+  imports: [
+    CommonModule,
+    TranslatePipe,
+    ReactiveFormsModule,
+    EditorComponent,
+    ButtonComponent,
+    InputTextModule,
+  ],
+  providers: [TournamentDataService, TeamDataService],
+  templateUrl: './page-manage-tournament.component.html',
+  styleUrl: './page-manage-tournament.component.less',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PageManageTournamentComponent implements OnInit, OnDestroy {
+  private readonly tournamentDataService = inject(TournamentDataService);
+  private readonly teamsDataService = inject(TeamDataService);
+  public readonly tournament$ = this.tournamentDataService.tournamentDetails$;
+  public readonly teams$ = this.teamsDataService.teams$;
+
+  protected readonly ButtonColorEnum = ButtonColorEnum;
+  protected readonly ButtonTypeEnum = ButtonTypeEnum;
+  protected readonly ButtonFunctionTypeEnum = ButtonFunctionType;
+  protected readonly EmailRecipientEnum = EmailRecipientEnum;
+  private readonly destroy$ = new Subject<void>();
+  protected tournamentId: number | null = null;
+
+  public readonly form = new FormGroup<{
```

Comment:
Warum erstellst du kein extra Model für das Form?

Reply:
ja gute Frage. Schreib ich in die Nacharbeiten :)